net/znc/Makefile

```diff
@@ -23,6 +23,7 @@ PKG_LICENSE_FILES:=LICENSE
 PKG_USE_MIPS16:=0
 PKG_BUILD_PARALLEL:=1
 
+PKG_CONFIG_DEPENDS := CONFIG_ZNC_ICU
 
```

Comment:
Nothing (directly) to do with the option showing up in  menuconfig, this now works. The detection that this symbol changed is broken. To test this:

1. `make package/znc/clean`
2. set ZNC_ICU to n
3. `make package/znc/compile` (should compile it)
4. change ZNC_ICU to y
5. `make package/znc/compile` (should recompile it, but currently it doesn't)